helix-core/src/transaction.rs

```diff
@@ -758,7 +758,7 @@ mod test {
 
     #[test]
     fn combine_with_utf8() {
-        const TEST_CASE: &'static str = "Hello, これはヒレクスエディターです！";
+        const TEST_CASE: &'static str = "Hello, これはヘリックスエディターです！";
```

Comment:
@CBenoit Err, why isn't it `へニックス`
```suggestion
        const TEST_CASE: &'static str = "Hello, これはへニックスエディターです！";
```

Reply:
ニ is ni, リ is ri 😅